tools/resourcedocsgen/pkg/lookup.go

```diff
@@ -189,6 +190,7 @@ var TitleLookup = map[string]string{
 	"opsgenie":                            "Opsgenie",
 	"packet":                              "Packet",
 	"pagerduty":                           "PagerDuty",
+	"pgedge":                              "pgEdge",
```

Comment:
`TitleLookup` is being deprecated in favor of the `"displayName"` field in the provider schema. If you set `tfbridge.ProviderInfo.DisplayName`, then it will flow through.

We are not adding new packages to `lookup.go`.